domains/v.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "leofelix908",
+    "email": ""
+  },
+  "record": {
+    "CNAME": "v.is-a.dev"
```

change this to `xproject-xi.vercel.app`

```suggestion
    "CNAME": "xproject-xi.vercel.app"
```

---

i have updated it was typo